### ISSUE TITLE
Add new issue button not showing on iPhone X

### ISSUE THREAD

**Issue by csletten** [CodeHubApp/CodeHub#435]

**Comment by garrettw** [comment]
Could you post a screenshot of the page where you're expecting to see it?

**Comment by csletten** [comment]
![d83f16f7-8227-4c4e-8518-4c650425a23a](https://user-images.githubusercontent.com/35894685/35553471-55399f68-0598-11e8-91a9-666fca176a38.png)

**Comment by toppleone** [comment]
5山，奴是aa邑 我是be划

从我的 Apple Watch 发送

在 2018年1月30日，下午3:35，Christian Sletten <notifications@github.com> 写道：

**Comment by tecollo** [comment]
Hey ye o nasıl gidiyor dostum hava nasıl oralarda 

Vedat duymaz  gönderdi

toppleone <notifications@github.com> şunları yazdı (30 Oca 2018 10:49):

**Comment by garrettw** [comment]
@csletten What is "Søk" -- a user? A repo? Knowing this might help me to know how you got to this screen.
Forgive me for asking so many questions, but you didn't give much detail.
Also, I have an iPhone X and I've not seen this issue, which is why I'm trying to reproduce it on my own device.

**Comment by garrettw** [comment]
@csletten How exactly did you get to that screen? It looks like you came from a _Søk_ (search) page but which one? I can tell that the issues being shown in your screenshot are those from your own repo, "Calculator", but when I go to your repo and then tap Issues, I see a slightly different screen.

**Comment by csletten** [comment]
The reason Søk (search) is showing is because the app was opened from Spotlight search in iOS. I am viewing issues in my own Repo. The issue doesn't appear when viewing issues in other repos not belonging to me.
For comparison:
![img_0515](https://user-images.githubusercontent.com/35894685/35572975-1dab4342-05d6-11e8-8b8d-4defec3b1aa0.PNG)
![img_0516](https://user-images.githubusercontent.com/35894685/35573016-2f0198a8-05d6-11e8-8444-5034c404fcc7.PNG)

**Comment by garrettw** [comment]
@thedillonb Why are his issues sorted in reverse order, and below a heading that says "Created 0 to 5 days ago"? As his top left corner shows a hamburger menu rather than a Back arrow, I'm inclined to think he's viewing this from the Issues tab in the menu -- except when I do that on my phone, the headings I see list which repo the issues below belong to, not how long ago they were created.

**Comment by garrettw** [comment]
@csletten If you are indeed viewing this using the Issues tab in the left-side menu, that is why there is no "+" button to create an issue; you may not be in any individual repo context to begin with.

**Comment by thedillonb** [comment]
Yeah, I'm also perplexed here. So the reason you're not seeing the "add new issue" button is because it's the issues view from the slide out menu - which is issues from every repository similar to: https://github.com/issues. 

The fact that the headings are times and not repository names are really weird. That's definitely a bug. It would seem at some time you went into `Custom` and created a filter that was ordered by field: "Created" (or "Updated"). But the fact you have "Open" definitely suggests a bug.

**Comment by csletten** [comment]
I see, sorry for the misunderstanding, I was accessing issues from the menu. 4 presses to add an issue just seems a bit excessive considering I currently only have one repo. What if the + button allowed you to select which repo you want to add an issue to?

**Comment by thedillonb** [comment]
@csletten I can definitely see that being a good addition!

**Comment by garrettw** [comment]
Yeah, that's not the worst idea, but @csletten keep in mind that most people -- well at least most people I've seen -- tend to own more than one repo.

**Comment by csletten** [comment]
Lastly, I notice a minor UI issue when using the X in Landscape while viewing issues. The separator between the issue no. and title seems to be offset.
![img_0517](https://user-images.githubusercontent.com/35894685/35573785-51e30616-05d8-11e8-8fd3-d3791a66e8e8.PNG)

**Comment by garrettw** [comment]
Can confirm. This happens on both the main Issues tab as well as the issues under a repo.

**Comment by thedillonb** [comment]
@csletten wow, that looks atrocious! I gotta give the issue view some love - maybe even rework the table cells to be more useful in landscape or iPad.
